tensor_test.go

```diff
@@ -144,3 +146,20 @@ func TestTensorPinMemory(t *testing.T) {
 		assert.Equal(t, " 1  2\n 3  4\n[ CPUFloatType{2,2} ]", b.String())
 	}
 }
+
+func TestTensorGC(t *testing.T) {
+	torch.GC()
+	runtime.LockOSThread()
```

Comment:
We have called `runtime.LocalOSTread()` function at  https://github.com/wangkuiyi/gotorch/blob/develop/device.go#L89 , can we remove this line?

Reply:
No. `go test` runs test cases in separate goroutines. The call of `runtime.LocalOSThread` only locks the *main goroutine*.